### ISSUE TITLE
filename contains link metadata when downloading a file

### ISSUE THREAD

**Issue by individual-it** [owncloud/web#6122]
no sure if this is a web issue or an ocis issue

### Steps to reproduce
1. run web from the current ocis master branch
2. upload a file
3. download a file and check the proposed filename

### Expected behaviour
filename should be the same as in the web view

### Actual behaviour
![image](https://user-images.githubusercontent.com/2425577/145516113-caa72357-08e9-49b9-a6b4-1260647fa882.png)

### Environment general
**Operating system**:

**Backend (ownCloud Core or Infinite Scale)**:
ocis

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Environment ownCloud Infinite Scale
**Version:**
current master
**Updated from an older Infinite Scale or fresh install:**
fresh
**Where did you install Infinite Scale from:**
git

#### Browser log
```
Content-Disposition | attachment;   filename*=UTF-8''file.txt?OC-Algo=PBKDF2%2F10000-SHA512&OC-Credential=admin&OC-Date=2021-12-10T04%3A12%3A15.278Z&OC-Expires=1200&OC-Verb=GET;   filename="file.txt?OC-Algo=PBKDF2%2F10000-SHA512&OC-Credential=admin&OC-Date=2021-12-10T04%3A12%3A15.278Z&OC-Expires=1200&OC-Verb=GET"
```

**Comment by pascalwengerter** [comment]
Duplicate of https://github.com/owncloud/ocis/issues/2856, fixed with the reva update yesterday :) (needs an oCIS commit ID bump which is blocked by broken/flaky smoke tests). Closing here nonetheless
